docs/pages/advanced/starknet.md

```diff
@@ -0,0 +1,223 @@
+## Verifying Stone proofs on Starknet
+
+This guide provides detailed instructions on creating and verifying Stone proofs on Starknet as well as how to deploy integrity contracts. Follow the steps below to ensure a smooth verification process.
+
+## Prerequisites
+
+Before you begin, ensure you have the following tools and dependencies installed these are outlined here [Integrity prerequisites](https://github.com/HerodotusDev/integrity?tab=readme-ov-file#prerequisites).
+
+- [Rust](https://www.rust-lang.org/tools/install)
+- [Scarb](https://docs.swmansion.com/scarb/download.html)
+- [Starknet Foundry](https://github.com/foundry-rs/starknet-foundry?tab=readme-ov-file#installation)
+
+Make sure to follow the next usage instructions as referenced [here](https://github.com/dipdup-io/stone-packaging?tab=readme-ov-file#usage-instructions).
+
+### Download Binaries for x86_64
```

Comment:
Let's replace it with a relative link to the Installation Guide (../install/binaries.md)